EXPO/native-base-theme/components/Body.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import variable from './../variables/platform';
+
+export default (variables /*: * */ = variable) => {
+  const bodyTheme = {
```

Comment:
directly return from here, no need to create variable.

Reply:
Above code is in Native base theme third party Library.